### PR TITLE
version api

### PR DESCRIPTION
**Pull request by Lucasmiguelmac**
Set api version by wrapping urlpatters with `"v1"`.

### CODE REVIEW

File: terra/terra/urls.py [hunk 1]
```diff
@@ -126,11 +126,15 @@
...
-urlpatterns += [path("credits/", include("credits.urls"))]
-urlpatterns += [path("admin/django-rq/", include("django_rq.urls"))]
-urlpatterns += [path("quotas/", include("quotas.urls"))]
-urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+urls_list += docs_urls
+urls_list += [path("storage/", include("storage.urls"))]
+urls_list += [path("tasks/", include("tasks.urls"))]
+urls_list += [path("estimators/", include("estimators.urls"))]
+urls_list += [path("credits/", include("credits.urls"))]
+urls_list += [path("admin/django-rq/", include("django_rq.urls"))]
```
Comment (munshkr) [inline terra/terra/urls.py:134]: Let's leave this outside the `/v1` namespace. This is an endpoint related to the django admin, not the API.

File: terra/terra/urls.py [hunk 2]
```diff
@@ -126,11 +126,15 @@
...
+urls_list += [path("quotas/", include("quotas.urls"))]
+urls_list += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (munshkr) [inline terra/terra/urls.py:136]: This also should be outside the `/v1` prefix, it's used by the admin I think.

**Review by munshkr — CHANGES_REQUESTED** [review]

File: terra/terra/urls.py (hunk 1 above)
Comment (Lucasmiguelmac) [reply terra/terra/urls.py:134]: Ok. You mean line 134 or all this 4 lines?

File: terra/terra/urls.py (hunk 2 above)
Comment (Lucasmiguelmac) [reply terra/terra/urls.py:136]: Ok. You mean line 136 (& 134) or all this 4 lines?

Comment (munshkr) [reply terra/terra/urls.py:136]: Yes sorry, I meant only 136 and 134 (`/admin/django-rq/` and the static endpoint)

**Comment by munshkr** [comment]
I think the admin should be left out as well. It should be accessible at `/admin/` not `/v1/admin/`.

**Pull request merged**